src/pages/SignInOrUp/SignInOrUp.tsx

```diff
@@ -235,7 +236,7 @@ export const SignInOrUp: React.FC = () => {
         </Grid>
       </Grid>
 
-      <Grid item xs className={classes.imageSideContentContainer} />
+      <Grid item xs className={classes.imageSideContentContainer} style={{ backgroundImage: `url("${custom?.images?.auth || rocketMan}")` }} />
```

Comment:
If we are to provide the backgroundImage property here (i.e., in an inline style), we should delete it from the `imageSideContentContainer` class. If you find no need for the `background` property in said class, please remove it.